research/00-introduction.tex

```diff
@@ -2,6 +2,30 @@ \section*{Introduction}
 \label{sec:introduction}
 \addcontentsline{toc}{section}{Introduction}
 
+Spoofax is a language workbench developed by TU Delft over the course of several
+years. During those years it has grown to `a platform for developing textual
```

Comment:
If this is you paraphrasing someone, 1) use "", 2) italics and 3) most importantly, cite!